### PR TITLE
module/ideas: preparing structure and aligning to wireframes (wip)

### PR DESCRIPTION
**Pull request by khamui**
Dividing elements into container and presentationals.
Prestyling (to be changed) according to structure of wireframes.
ToDo: Refactoring, Icon handling, styling to meet designs (wip).

### CODE REVIEW

File: containers/Ideas/IdeasListItem.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { styles } from './IdeasListItem.styles';
+
+import { ButtonCounter } from '../../components/buttonCounter';
+
+export const IdeasListItem = (props) => {
+  console.log(props);
```
Comment (Rineee) [inline containers/Ideas/IdeasListItem.js:8]: Could you remove this?

**Review by Rineee — CHANGES_REQUESTED** [review]
Really cool!
Could you maybe somehow squash your commits, at least the first and second? So the mocked data is gone :)
And there are still some files without newlines at the end, could you add them, please?

The rest looks good to me, so we can go on from there!

File: containers/Ideas/IdeasListItem.js (hunk 1 above)
Comment (khamui) [reply containers/Ideas/IdeasListItem.js]: ahh, thanks..maybe i'll add another eslint rule.

**Comment by khamui** [comment]
okay now should be fixed with linting rules and squashed. additionally i removed the sample data.

**Review by Rineee — APPROVED** [review]

**Pull request merged**